packages/plugin-kit/src/createSliceMachinePluginRunner.ts

```diff
@@ -48,6 +48,9 @@ export const REQUIRED_ADAPTER_HOOKS: SliceMachineHookTypes[] = [
 	"custom-type:asset:read",
 	"custom-type:asset:delete",
 	"custom-type-library:read",
+	"project:init",
+	"project:environment:read",
+	"project:environment:update",
```

Comment:
Have we considered possibly making those hooks optional in order not to raise the bar to implement new minimal adapters?

Reply:
They are optional now. 🙂 However, with the possible introduction of dev-only environments, we may want to promote it to a required hook in the future. Dev environments may be a feature we want to push developers to use.